patches/testem+3.10.0.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/testem/lib/server/index.js b/node_modules/testem/lib/server/index.js
+index a1baf70..e3545f7 100644
+--- a/node_modules/testem/lib/server/index.js
++++ b/node_modules/testem/lib/server/index.js
+@@ -177,7 +177,7 @@ class Server extends EventEmitter {
+       }
+       next();
+     });
+-    app.use(express.static(`${__dirname}/../../public`));
++    app.use('/testem', express.static(`${__dirname}/../../public/testem`, {fallthrough: false}));
```

Comment:
Testem uses an express [`serve-static`](https://github.com/expressjs/serve-static) middleware to serve the assets. It's mounted on the app root, meaning every request is first processed with the `express.static` middleware and only after that with the middlewares defined in the config, e.g., proxy middleware.

This ends up in a bunch of extra `fs.stat` calls which bloat the server response time. You can check that by enabling the `DEBUG=send` environment variable and making a request to the server.

I made a PR that fixes this behavior in the `testem` https://github.com/testem/testem/pull/1615

Reply:
I've removed the patch and updated `testem` to v3.10.1 which includes the fix.